shopfloor_packing_info/models/shopfloor_packing_info.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Camptocamp SA (https://www.camptocamp.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+from odoo import fields, models
+
+
+class ShopfloorPackingInfo(models.Model):
+    _name = "shopfloor.packing.info"
+    _description = "Shopfloor Packing Information"
+
+    name = fields.Char("Name", required=True)
+    text = fields.Text("Text", required=True)
+    active = fields.Boolean("Active", default=True)
+
+    def unlink(self):  # pylint: disable=W8106
+        """Archive, not unlink"""
+        self.write({"active": False})
```

Comment:
I'm not fund of this:

* if the record has been used, you'll have an error message as there is `ondelete='restrict'`, then you can archive it
* if the record has not been used, then I don't see the problem of deleting it (say you just created one by error)
* it goes against the conventions in other models, so is misleading for users

Reply:
I agree. This was not a good idea...